tests/data_files/fixed/crl-ec-sha1.pem

```diff
@@ -0,0 +1,10 @@
+-----BEGIN X509 CRL-----
```

Comment:
I don't understand what “fixed” is supposed to mean in the directory name. Please describe this in a readme file. I'm not sure if “fixed” is a good name.

Reply:
I have re-written the history and this comment might be out of data